build_helpers/test-globals.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+var jsdom = require('jsdom')
```

Comment:
That's what I get for copying it from somewhere else.  Sure would be nice to have ESLint set up for this project.  Any interest in a PR doing so?